### DISCUSSION TITLE
Why do you keep 4.0 version?

### DISCUSSION THREAD

**Discussion by cmdntd** [socioboard/Socioboard-5.0#274] [General]
Sorry. I do not understand.
Why should I use 4.0 when you had 5.0 release? Why do you not merge them?

**Comment by sureshbabu1995** [comment]
Hi, @cmdntd 
That's a good question, but see the 5.0 is not a stabled one and it's not as expected, we're working on it,

I want the 5.0 as a Major and we want to introduce it will all team

So, if any person wants to go with an old one then they can.

We'll release updates soon. But the master code is complete of 5.0 code.

**Reply by TechnologyClassroom** [reply]
This response does not make sense as many 4.0 issues were closed on July 23rd [announcing the update to 5.0](https://github.com/socioboard/Socioboard-5.0/issues/172#issuecomment-885652637).  If 5 is not stable, those issues should not have been closed.  If 5 is not stable, why is the repository named `Socioboard-5.0`?

Why is the version in the repository name to begin with?  Typically version information would be kept in `README.md` and `CHANGLOG.md` files and the name of the repository would just be `Socioboard`.

**Comment by cmdntd** [comment]
I'm waiting socioboard from version 3.0 for a more stable one after many failed test. I want to try in 5.0 for test. Could you help more in these?

1. I want to keep upgrade smoothly after the first installation. So, it seems complicated because there are 2 version like 4.0 and 5.0 with many difference. I just want to start with newest like 5.0 for better upgrade.
2. I want to send more question and issues for contribution. But github should be just only for bug or fix issues. Where could I send you question for installation/usage issues?
3. I'd love use socioboard in docker. Could you build official 5.0 docker one for this?

**Comment by sureshbabu1995** [comment]
Hi @cmdntd 

First, Thank for your interest, all the 3 queries can be done..

1. Try with 5.0 we're stable on it, and we're bringing many updates to it, and you can get better support for smooth upgrades
2.  You can send mail to me - sureshbabu@globussoft.in or you can join in Telegram (https://t.me/sureshbabu_globussoft) or even in Skype (https://join.skype.com/invite/codooErFDBEb)
3. Yes, Docker might take time, but yes soon we'll be adding docker to it.
